ceph/check.py

```diff
@@ -26,7 +25,7 @@ class Ceph(AgentCheck):
     DEFAULT_CEPH_CLUSTER = 'ceph'
     NAMESPACE = 'ceph'
 
-    def _collect_raw(self, ceph_cmd, ceph_cluster, instance):
+    def _get_ceph_args(self, ceph_cmd, ceph_cluster, instance):
```

Comment:
`_collect_raw` was mocked in the tests so this change breaks the tests. I agree that `_collect_raw` might not be the most accurate of tests, and since we're prepending the `_` it's probably acceptable to change the method name. Could you please fix the tests as well. See here: https://github.com/DataDog/integrations-core/blob/master/ceph/test_ceph.py#L22 (note there are multiple occurrences in that test file). Once that's fixed, we'll know for sure if there's anything that might've broken for older ceph's.